### ISSUE TITLE
Invalid access attempt to a computed property on addModel()

### ISSUE THREAD

**Issue by joelmoss** [ctrlplusb/easy-peasy#546]
In https://github.com/ctrlplusb/easy-peasy/blob/v3.4.0/src/computed-properties.js#L27 a warning is thrown `Invalid access attempt to a computed property`, which unfortunately swallows the error thrown by redux.

I'm encountering this warning when calling `addModel`, but am unsure why. Can you elaborate as to what the underlying issue is and why a warning is thrown instead of the error?

thx

**Comment by ctrlplusb** [comment]
Hey @joelmoss 👋

Thanks for the report!

**Comment by joelmoss** [comment]
Ok, so after spending way too much time hunting this one down, it came down to two bugs, neither of which were due to easy-peasy. So yay! 🥳

One bug was in my code, and the other was caused by Redux dev tools browser extension - I ended having to simply disable the extension.

However, the try/catch that exists at https://github.com/ctrlplusb/easy-peasy/blob/v3.4.0/src/computed-properties.js#L23-L30 did not help me here, as it hides the underlying issue.

Is there a reason why this try/catch block is here?
